haiku-apps/nanodot/nanodot-1.1b.recipe

```diff
@@ -0,0 +1,46 @@
+SUMMARY="A minimal pattern-based MIDI sequencer"
+DESCRIPTION="nanodot is a rather minimal pattern-based MIDI sequencer \
+originally written for BeOS R3. Its main purpose in life was/is to teach me \
+(and hopefully others) the basics of writing MIDI apps for BeOS; it makes for \
+a nice drum-programming 'scratchpad', and is a generally entertaining way to \
+waste a few hours."
+HOMEPAGE="https://github.com/HaikuArchives/NanoDot"
+COPYRIGHT="1999 Eric Moon"
+LICENSE="unknown"
+REVISION="1"
+srcGitRev="e201c46fdcb97ee919393f0f6eb8106eb51a0797"
+SOURCE_URI="https://github.com/HaikuArchives/NanoDot/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="f12538e765a13f30d5dd2ef344da14b682a348fd48ffa2e2e5819c0d6722f19c"
+SOURCE_FILENAME="NanoDot-$portVersion-$srcGitRev.tar.gz"
+SOURCE_DIR="NanoDot-$srcGitRev"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	nanodot = $portVersion
+	app:nanodot = $portVersion
+	"
+REQUIRES="
+	haiku
+	"
+
+BUILD_REQUIRES="
+	haiku_devel
+	"
+BUILD_PREREQUIRES="
+	makefile_engine
+	cmd:g++
+	cmd:make
+	"
+
+BUILD()
+{
+	make OBJ_DIR=objects
+}
+
+INSTALL()
+{
+	install -d $appsDir
+	install -t $appsDir objects/NanoDot
+	addAppDeskbarSymlink $appsDir/NanoDot NanoDot
```

Comment:
```diff
-	addAppDeskbarSymlink $appsDir/NanoDot NanoDot
+	addAppDeskbarSymlink $appsDir/NanoDot
```
(The last argument is not required if the name is the same, so we prefer to not repeat it.)